crates/actor/src/module/random.rs

```diff
@@ -0,0 +1,70 @@
+use super::host;
+
+use core::iter;
+
+use serde::Serialize;
+use uuid::Uuid;
+
+/// Return a cryptographically-secure pseudo-random [`u64`] value.
+pub fn get_random_u64() -> u64 {
+    u64::from(random32().reverse_bits()) | u64::from(random32())
+}
+
+/// Return `len` cryptographically-secure pseudo-random bytes.
+pub fn get_random_bytes(n: u64) -> Vec<u8> {
+    let n = n.try_into().expect("too many bytes requested");
+    iter::repeat_with(random32)
+        .map(u32::to_ne_bytes)
+        .flatten()
+        .take(n)
+        .collect()
+}
+
+/// Return a 128-bit value that may contain a pseudo-random value.
```

Comment:
Nit: I assume this function by definition has to return the two `u64`s rather than a `u128`. If that assumption is correct, it might be worth documenting that this returns two u64s that can be combined into a u128

Reply:
This is duplicate of what `wit-component` generates for component-model case, (more recent) source: https://github.com/WebAssembly/wasi-random/blob/257e4d73b0e3d034d3c147876d51d1b3db92b17b/wit/insecure-seed.wit#L6-L16